.lycheeignore

```diff
@@ -0,0 +1 @@
+// add some link here that you dont want link-checker ci to check
```

Comment:
Why this file in the root folder? Why not in the same folder than the workflow code?